### PR TITLE
ugly solution 4cpu

### PR DESCRIPTION
**Pull request by Masclins**
I found a way to make up to 4 CPUs work on this.
It's really ugly though.
I opt for this because the alternative of making the probability of each prize calculated by a different cpu didn't seem to save any time.

### CODE REVIEW

**Comment by Masclins** [comment]
@kenan-rhoton I'd really appreciate a review here :)

File: test/rtp/rtp_test.clj [hunk 1]
```diff
@@ -5,19 +5,22 @@
...
+                      :suits  [0 1 2 3 1]
+                      :times  1}
+                     {:values [0 9 10 11 12]
+                      :suits  [0 0 0 0 0]
+                      :times  2}
+                     {:values [0 0 0 0 1]
+                      :suits  [0 1 2 3 1]
+                      :times  1}])))
+    (is (= (/ 86698 61617)
+(time      (rtp/rtp [{:values [0 11 10 9 8]
```
Comment (Masclins) [inline test/rtp/rtp_test.clj:21]: I left it here because it lets me see if future perfomance changes have any incidence.

**Comment by Masclins** [comment]
Updated to only 2 CPUs

**Comment by kenan-rhoton** [comment]
I can take a look at stuff like this next week

**Pull request merged**